src/code/z_player_lib.c

```diff
@@ -796,9 +833,11 @@ Gfx* gPlayerWaistDLs[2 * PLAYER_FORM_MAX] = {
     gLinkZoraWaistDL,        gLinkDekuWaistDL,        gLinkDekuWaistDL,  gLinkHumanWaistDL, gLinkHumanWaistDL,
 };
 
-Gfx* gPlayerHandHoldingShields[PLAYER_SHIELD_MAX - 1][2] = {
-    { gLinkHumanRightHandHoldingHylianShieldDL, gLinkHumanRightHandHoldingHylianShieldDL },
-    { gLinkHumanRightHandHoldingMirrorShieldDL, gLinkHumanRightHandHoldingMirrorShieldDL },
+Gfx* gPlayerHandHoldingShields[2 * (PLAYER_SHIELD_MAX - 1)] = {
+    gLinkHumanRightHandHoldingHylianShieldDL,
+    gLinkHumanRightHandHoldingHylianShieldDL,
```

Comment:
The shield is called the "Hero's Shield" in-game:
![image](https://user-images.githubusercontent.com/12245827/232888210-3cae31be-c928-4c15-9c6e-b5b3ac689edb.png)